src/lib/estimates.js

```diff
@@ -9,17 +8,17 @@
  * @return {Number} Estimated number of days until the next purchase
  */
 const calculateEstimate = (lastEstimate, latestInterval, numberOfPurchases) => {
-    if (numberOfPurchases >= 2) {
-        if (isNaN(lastEstimate)) {
-            lastEstimate = 14;
-        }
-        let previousFactor = lastEstimate * numberOfPurchases;
-        let latestFactor = latestInterval * (numberOfPurchases - 1);
-        let totalDivisor = numberOfPurchases * 2 - 1;
-        return Math.round((previousFactor + latestFactor) / totalDivisor);
-    } else {
-        return latestInterval;
+  if (numberOfPurchases >= 2) {
+    if (isNaN(lastEstimate)) {
+      lastEstimate = 14;
     }
+    let previousFactor = lastEstimate * numberOfPurchases;
+    let latestFactor = latestInterval * (numberOfPurchases - 1);
+    let totalDivisor = numberOfPurchases * 2 - 1;
+    return Math.round((previousFactor + latestFactor) / totalDivisor);
+  } else {
+    return latestInterval;
+  }
```

Comment:
@macosta08 and @CrisRuedaP, very clear code, just a small detail, maybe change the `let` to `const`, otherwise very good, great job.

Reply:
Hi @Lukaina 

This script was made by collab and provided in the issue 

![image](https://user-images.githubusercontent.com/70062856/106811017-bb839b00-663b-11eb-924b-0046ff20df0c.png)

Reply:
@macosta08 Hi,
Oh I see, sorry, I didn't notice that, perfect. Great job!